collection-scripts/gather_etcd

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+BASE_COLLECTION_PATH="/must-gather"
+ETCD_LOG_PATH="${BASE_COLLECTION_PATH}/etcd_info"
+
+RUNNING_ETCD_POD=$(oc get pod -n openshift-etcd --no-headers|grep Running| grep -o -m 1 '\S*etcd\S*')
+
+function ocp4etcdctl {
+    oc -n openshift-etcd exec -c etcdctl ${RUNNING_ETCD_POD} -- etcdctl $@
+}
+
+if [[ -z $RUNNING_ETCD_POD ]];then
+    echo "ERROR: No running etcd pods found" >&2
+    exit 1
+fi
+
+mkdir -p ${ETCD_LOG_PATH}
+
+echo "INFO: Starting getting etcd information"
+
+# member list
```

Comment:
Should we run all of these in parallel?

Reply:
All those commands run very quickly (except for the check perf), so this is why I didn't make this parallel

Reply:
It's still ~8 `oc exec` calls. Depending on server load this could be cumbersome. Threading this lets all of this run at the same time and simply stop when completed, vs each call getting hung up and stalling the collection.

Reply:
Fair enough. I'll use the same parallelism scheme used in other gather scripts